### ISSUE TITLE
Unit tests for all `dev` strategies

### ISSUE THREAD

**Issue by Nullably** [CoinAlpha/hummingbot#3267]
After some discussion with @Nullably
This ticket involves the including unit test cases for the following strategies:
1. dev_0_hello_world
2. dev_1_get_order_book
3. dev_2_perform_trade
~~4. dev_4_twap~~
~~5. dev_5_vwap~~
6. dev_simple_trade

Additional Tasks:
- Refactoring the strategies to utilise the Python based architecture.

**To-Do List**
- [x] Included Unit tests for dev_0_hello_world
- [x] Included Unit tests for dev_1_get_order_book
- [x] Included Unit tests for dev_2_perform_trade
- [x] Included Unit tests for dev_simple_trade

**Comment by CrimsonJacket** [comment]
As part of this maintenance sprint, @Nullably and I believe that it will be beneficial to refactor some of the simpler `dev` strategies to extend the Python-based `StrategyPyBase` class over the Cython-based `StrategyBase` parent class. As such I have created another ticket for this refactor.

Additionally, after a pair programming session with @Nullably, it is evident that some of the dev strategies, `dev_4_twap` and `dev_5_vwap` is not working as intended. Hence, I've created new tickets to fix and include unit tests those strategies.

Below are the new tickets created for the aforementioned issues.
1. Refactor 'dev' strategies to use Python-based architecture #3349
2. Fix `dev_4_twap` and `dev_5_vwap` strategy #3350
